databricks/koalas/tests/test_series.py

```diff
@@ -242,6 +242,32 @@ def test_sort_values(self):
         self.assert_eq(ks.sort_values(inplace=True), ps.sort_values(inplace=True))
         self.assert_eq(repr(ks), repr(ps))
 
+    def test_sort_index(self):
+        ps = pd.Series([2, 1, np.nan], index=['b', 'a', np.nan], name='0')
+        ks = koalas.from_pandas(ps)
+
+        # Assert invalid parameters
+        self.assertRaises(ValueError, lambda: ks.sort_index(axis=1))
+        self.assertRaises(ValueError, lambda: ks.sort_index(level=42))
+        self.assertRaises(ValueError, lambda: ks.sort_index(kind='mergesort'))
+        self.assertRaises(ValueError, lambda: ks.sort_index(na_position='invalid'))
+
+        # Assert default behavior without parameters
+        self.assert_eq(repr(ks.sort_index()), repr(ps.sort_index()))
+        # Assert sorting descending
+        self.assert_eq(repr(ks.sort_index(ascending=False)), repr(ps.sort_index(ascending=False)))
+        # Assert sorting NA indices first
+        self.assert_eq(repr(ks.sort_index(na_position='first')),
+                       repr(ps.sort_index(na_position='first')))
+        # Assert sorting inplace
+        self.assert_eq(ks.sort_index(inplace=True), ps.sort_index(inplace=True))
```

Comment:
nit: this is only comparing `None` vs. `None`.

Reply:
I've replaced `assert_eq` with `assertEquals` in a64999e. However, comparing `None` with `None` is still necessary to make sure the Koalas implementation has the same return value as the pandas equivalent.